lib/KeyboardAwareMixin.js

```diff
@@ -35,19 +47,25 @@ const KeyboardAwareMixin = {
       keyboardSpace: keyboardSpace,
     })
     // Automatically scroll to focused TextInput
-    const currentlyFocusedField = TextInput.State.currentlyFocusedField()
-    if (!currentlyFocusedField) {
-      return
-    }
-    UIManager.viewIsAncestorOf(
```

Comment:
Hi,

With ("react-native": "^0.28.0") this will cause an error:

```
ExceptionsManager.js:70 Unhandled JS Exception: 
  _reactNative.UIManager.viewIsAncestorOf is not a function
```